metis-enrichment/metis-enrichment-client/src/main/java/eu/europeana/enrichment/utils/EntityMergeEngine.java

```diff
@@ -334,15 +343,24 @@ private static TimeSpanType convertTimeSpan(Timespan timespan) {
     return timeSpanType;
   }
 
+  private static eu.europeana.metis.schema.jibx.Organization convertOrganization(
+      Organization organization) {
+    final eu.europeana.metis.schema.jibx.Organization organizationType = new eu.europeana.metis.schema.jibx.Organization();
+    organizationType.setAbout(organization.getAbout());
+    organizationType.setPrefLabelList(
+        ItemExtractorUtils.extractLabels(organization.getPrefLabelList(), PrefLabel::new));
+    return organizationType;
+  }
```

Comment:
Is this really all we need from organizations? Or are we missing fields?

Reply:
That's all we need for the time being.